README.md

```diff
@@ -0,0 +1,179 @@
+
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://git-scm.com/">HTML and CSS</a></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Key Feature1:]**  Code Correction is done by Linter
+- **[Key Feature2:]**  Applied Correct Githubflow and documented in professional way
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- vscode.
+
+- git.
+
+- set up an index.html document.
+- set up a style.css document.
+
+### Setup
+
+Clone this repository to your desired folder:
+- Navigate to the directory where you want clone this project in your computer
+`$ cd Desktop/`
+`$ git clone https://github.com/rahimialijan/Hello-Microverse.git`
+
+### Install
+
+Install this project with this command:
+
+`$ npm install`
+
+### Usage
+
+To run the project, execute the following command by opening the vscode with the command `code .`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Alijan Rahimi**
+
+- GitHub: [@Alijan-Rahimi](https://github.com/rahimialijan)
+- Twitter: [@AlijanRahimi10](https://twitter.com/AlijanRahimi10)
+- LinkedIn: [Alijan-Rahimi](https://www.linkedin.com/in/alijan-rahimi-18389ab3)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [1] **[ Setting Up A Professional Github Repository.]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+ If you like this project, give it a star ⭐
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for setting up this project for student
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+
+- **[How is learning in Microverse program? ]**
+
+  - [I have to say , I really enjoy and get a lot of experience.]
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] I can see that you made some changes required in the previous review , that good job so far but I just noticed that your `M.I.T` Licence still need to be fixed , as required in the [Previous review](https://github.com/rahimialijan/Hello-Microverse/pull/1#discussion_r1143914670) , follow  the steps given by the previous reviewer and I'm sure you are going to make it , kindly fix this :point_down: 
![Screenshot from 2023-03-22 17-14-00](https://user-images.githubusercontent.com/88894525/226952605-f936311a-6ddf-402c-8189-496f3876f636.png)

Reply:
Hi Sir, I made the required changes for the readMe file. Please check it and let me know for next step